src/views/ModelPlan/TaskList/ITSolutions/SolutionImplementation/index.tsx

```diff
@@ -189,32 +200,82 @@ const SolutionImplementation = () => {
     return <NotFound />;
   }
 
+  const renderCancelCopy = () => {
+    if (isUpdatingStatus && fromSolutionDetails) {
+      return t('dontUpdateandReturnToSolutionDetails');
+    }
+    if (isUpdatingStatus) {
+      return t('dontUpdateandReturnToTracker');
+    }
+    return t('dontAdd');
+  };
+
   return (
     <>
-      <BreadcrumbBar variant="wrap">
-        <Breadcrumb>
-          <BreadcrumbLink asCustom={UswdsReactLink} to="/">
-            <span>{h('home')}</span>
-          </BreadcrumbLink>
-        </Breadcrumb>
-        <Breadcrumb>
-          <BreadcrumbLink
-            asCustom={UswdsReactLink}
-            to={`/models/${modelID}/task-list/`}
-          >
-            <span>{h('tasklistBreadcrumb')}</span>
-          </BreadcrumbLink>
-        </Breadcrumb>
-        <Breadcrumb>
-          <BreadcrumbLink
-            asCustom={UswdsReactLink}
-            to={`/models/${modelID}/task-list/it-solutions`}
-          >
-            <span>{t('breadcrumb')}</span>
-          </BreadcrumbLink>
-        </Breadcrumb>
-        <Breadcrumb current>{t('selectSolution')}</Breadcrumb>
-      </BreadcrumbBar>
+      {fromSolutionDetails ? (
+        <BreadcrumbBar variant="wrap">
+          <Breadcrumb>
+            <BreadcrumbLink asCustom={UswdsReactLink} to="/">
+              <span>{h('home')}</span>
```

Comment:
I think this can be refactored a bit.  We may want to look how EASI has created a much more condensed version of breadcrumbs.